### PR TITLE
Fix counties map/8

### PR DESCRIPTION
**Pull request by JasonHuang8**
Clicking a county on the map should lead to a list of representatives of that county with no errors now.

### CODE REVIEW

**Comment by zephthomas** [comment]
Nice, looks good! Approving but rspec failures need to be fixed before merge

File: app/javascript/packs/state_map_utils.js [hunk 1]
```diff
@@ -64,7 +64,11 @@ exports.setupEventHandlers = (stateMap) => {
     };
     const clickCallback = (elem) => {
         const countyFipsCode = elem.attr('data-county-fips-code');
-        window.location.href = `/state/${stateMap.state.symbol}/county/${countyFipsCode}`;
+        const state = stateMap.state.symbol;
+        const countyName = elem.attr('data-county-name');
+        const stateCounty = encodeURIComponent(`${countyName}, ${countyFipsCode}, ${state}`);
+        window.location.href = `/search/${stateCounty}`;
+        //window.location.href = `/state/${stateMap.state.symbol}/county/${countyFipsCode}`;
```
Comment (zephthomas) [inline app/javascript/packs/state_map_utils.js:71]: can probably remove this comment

**Review by zephthomas — APPROVED** [review]
Nice, looks good! Approving but rspec failures need to be fixed before merging.

File: app/javascript/packs/state_map_utils.js (hunk 1 above)
Comment (zephthomas) [reply app/javascript/packs/state_map_utils.js:71]: This is what is causing the lint error

Comment (JasonHuang8) [reply app/javascript/packs/state_map_utils.js:71]: gotcha

**Pull request merged**